x-pack/plugins/cases/server/client/cases/create.test.ts

```diff
@@ -18,6 +18,8 @@ import { createCasesClientMockArgs } from '../mocks';
 import { create } from './create';
 import { CaseSeverity, CaseStatuses, ConnectorTypes } from '../../../common/types/domain';
 
+import type { CaseCustomFields } from '../../../common/types/domain';
+
 describe('create', () => {
```

Comment:
I think I still have to change `update.test.ts`.

Reply:
Maybe we can add a test where you can update a custom field. If updating is not supported in this PR we can do it on another one.